### ISSUE TITLE
OverflowError: cannnot convert float infinity to integer in faultTypes.py Line 95

### ISSUE THREAD

**Issue by ElaineYao** [DependableSystemsLab/TensorFI#39]
I'm getting the following error when running **TensorFI/Tests/DNN-model/LeNet-mnist/LeNet.py**,
```
File "/home/xxx/TensorFI/faultTypes.py", line 95, in getBinary
      integer = bin(int(number)).lstrip("0b")

OverflowError: cannot convert float infinity to integer
```
I'm not sure whether there is infinity value and if so, should we truncate the value in the corresponding source code?

Thanks,

**Comment by zitaoc** [comment]
If it's a minor issue, and you think you can fix it; you can do so and send a pull request. 

Otherwise i'll need more details (e.g., how to reproduce) as I haven't seen this before.

**Comment by nniranjhana** [comment]
Fixed with #40
